src/components/Layout/Header/Header.tsx

```diff
@@ -79,37 +81,63 @@ export const Header: FunctionComponent<Props> = ({ minimal, colorTheme, navRef }
 
 const HEADER_CONTENT_THEME_CLASS: Record<
     HeaderColorTheme,
-    Record<'container' | 'item' | 'menu' | 'menuItem' | 'menuItemActive' | 'divider' | 'button' | 'panel', string>
+    Record<
+        | 'container'
+        | 'item'
+        | 'menu'
+        | 'menuItem'
+        | 'menuItemActive'
+        | 'divider'
+        | 'button'
+        | 'panel'
+        | 'icon'
+        | 'decoratedText'
+        | 'subText'
+        | 'arrowIcon',
+        string
+    >
 > = {
     white: {
-        container: 'bg-white',
-        item: 'text-black hover:bg-violet-200 focus:ring-black',
-        menu: 'bg-white ring-black',
-        menuItem: 'text-black',
-        menuItemActive: 'bg-violet-200',
+        container: 'bg-gray-50',
+        item: 'text-gray-500 hover:bg-gray-200 focus:ring-black',
```

Comment:
I've adjusted the hover colors and added a focus state in the Figma here: https://www.figma.com/design/o1QRtdQI0ozKq0n7ATrKlx/Marketing-DLS?node-id=13755-21325&m=dev&focus-id=14131-40010